deployment/helm/ditto/values.yaml

```diff
@@ -1674,6 +1687,16 @@ connectivity:
         expireAfterWrite: 4h
         # expireAfterAccess prolonged on each cache access by that duration
         expireAfterAccess: 2h
+    # signalEnrichment contains configuration regarding signal enrichment (retrieval of "extra" fields to e.g. enrich events)
+    signalEnrichment:
+      # cache holds the configuration of signal enrichment caching
+      cache:
+        # enabled whether signal enrichment caching (incl. smart updates updating the cache based on event data) should be enabled
+        enabled: true
+        # maximumSize how many things to cache in total on a single cluster node
+        maximumSize: 2000
```

Comment:
since this is the number of things and not a buffer shouldn't we increase this to match the 20 000 as configured in the connectivity.conf:
maximum-size = 20000
Or do we want it 2000... since 20 000 * 100kb would be 2 GB worth of things.
So maybe it makes more sense for a barebones default install to be 2000 instead of 20 000 :)